src/renderer_gl.cpp

```diff
@@ -1479,7 +1479,7 @@ namespace bgfx { namespace gl
 				, _internalFormat
 				, _width
 				, _height
-				, false
+				, true
```

Comment:
When we have multiple `GL_TEXTURE_2D_MULTISAMPLE` attachments this is required to be `true`, so that all sample locations match.
Realistically I don't know how common it is for sample locations to not match anyway, and I don't believe there is any penalty to doing so, so might as well have it on all the time for now.